### PR TITLE
[TG-133] Added ability to remove scenarios from list and form

### PR DESCRIPTION
**Pull request by Todmy**
- refactored mock server's scenario data
- added delete method to Scenario.service
- added get method and refactored getScenario method in Scenario.service
- added styles for scenario list
- added empty page for scenario list
- added and refactored unit tests for scenario list and details component

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/36" target="_blank">ReviewNinja

File: src/app/scenarios/scenarios-list/scenario-list.component.test.js [hunk 1]
```diff
@@ -30,25 +51,48 @@ describe('ScenarioListComponent', () => {
...
+            expect(sut.headers).toEqual(allowedHeaders);
         });
     });
 
-    describe('Open scenario', () => {
-        it('will open scenario by id', () => {
-            const id = 1;
-            const scenario = {id};
+    describe('#removeScenario', () => {
+        it('should be defined', () => {
```
Comment (antonSavk) [inline src/app/scenarios/scenarios-list/scenario-list.component.test.js:72]: this test is useless cause next test will fail in any case if there will be no "removeScenario" method. !fix

File: src/app/scenarios/Scenario.service.test.js [hunk 2]
```diff
@@ -115,4 +125,20 @@ describe('ScenarioService', () => {
             });
         });
     });
+
+    describe('Delete scenarios', () => {
+        it('should be defined', () => {
```
Comment (antonSavk) [inline src/app/scenarios/Scenario.service.test.js:130]: this test is useless cause next test will fail in any case if there will be no "removeScenario" method. !fix

File: src/app/scenarios/edit-scenario/edit-scenario.component.test.js [hunk 3]
```diff
@@ -10,19 +13,50 @@ describe('EditScenarioComponent', () => {
...
+    class ScenarioServiceMock {
+        set reqStatForTest(status) {
+            this._status = status;
+        }
+        getScenario() { return new MockPromise(true, scenario); }
+        updateScenario() { return new MockPromise(true); }
+        delete(data) {
+            const status = this._status || 200;
+            return {
+                subscribe(fn) {
```
Comment (antonSavk) [inline src/app/scenarios/edit-scenario/edit-scenario.component.test.js:25]: this test is useless cause next test will fail in any case if there will be no "removeScenario" method. !fix

**Pull request merged**